packages/runtime/src/slate/BlockPlugin/setBlockKeyForDevice.ts

```diff
@@ -20,7 +20,13 @@ export function setBlockKeyForDevice(
   for (const [node, path] of rootElements) {
     if (ElementUtils.isBlock(node)) {
       const deviceValues = node[key] ?? []
-      const currentDeviceValue = findDeviceOverride(deviceValues, deviceId, v => v)
+      // TODO: Get from registered breakpoints instead of the default breakpoints
```

Comment:
I think this can be fixed by modifying the plugin so that it takes the list of devices/breakpoints as an argument 🤔